.github/workflows/docker-image.yml

```
@@ -40,6 +40,10 @@ jobs:
             type=ref,event=tag
             type=ref,event=pr
             type=raw,value=build-${{ github.run_id }}
+            ${{ github.ref == 'refs/heads/main' && 'type=raw,value=mainnet' || '' }}
+            ${{ github.ref == 'refs/heads/main' && format('type=raw,value=mainnet-build-{0}', github.run_id) || '' }}
+            ${{ github.ref == 'refs/heads/testnet' && 'type=raw,value=testnet' || '' }}
```

```suggestion
            ${{ github.ref == 'refs/heads/main' && 'type=raw,value=testnet' || '' }}
```
We do not want to create separate branch "testnet" for this repo